app/models/message.rb

```diff
@@ -201,12 +201,17 @@ def update_recipients
   protected
 
   def can_send_message?(recipient)
-    if recipient.emails.present?
-      true
-    else
+    if recipient.receiver.suspended_or_scheduled_for_deletion?
+      logger.warn "Skipping message notification for Message(id:#{id}) because account is suspended."
+      return false
+    end
```

Comment:
I don't think we need the logging for this. Also I don't think we need logging for lack of emails. Especially not a `warn`. It's more of a debug message.

```suggestion
    return if recipient.receiver.suspended_or_scheduled_for_deletion?
```

Reply:
actually the whole method can be rewritten as 
```ruby
def can_send_message?(recipient)
  recipient.emails.present? && !recipient.receiver.suspended_or_scheduled_for_deletion? 
end
```